packages/gatsby-theme-docs/gatsby-config.js

```diff
@@ -203,6 +203,67 @@ module.exports = (themeOptions = {}) => {
           disable: process.env.ANALYZE_BUNDLE !== 'true',
         },
       },
+      {
+        resolve: 'gatsby-plugin-feed',
+        options: {
+          query: `
+          {
+            site {
+              siteMetadata {
+                title
+                description
+                siteUrl
+                site_url: siteUrl
+              }
+            }
+          }
+        `,
+          setup: (options) => {
```

Comment:
We can override the site title here to customize with title from `siteMetadata`.

But the link and description of the channel are lost once `setup` setup is implemented. Even when I tested with just returning the option parameter.

Reply:
You need to spread the options like this: https://github.com/gatsbyjs/gatsby/blob/055844312b8e681673fd453a8a556693eabb7ac1/packages/gatsby-plugin-feed/src/internals.js#L29-L39